src/component/axis/AxisBuilder.ts

```diff
@@ -269,18 +269,22 @@ const builders: Record<'axisLine' | 'axisTickLabel' | 'axisName', AxisElementsBu
         );
 
         const line = new graphic.Line({
-            shape: {
-                x1: pt1[0],
-                y1: pt1[1],
-                x2: pt2[0],
-                y2: pt2[1]
-            },
+            shape: graphic.subPixelOptimizeLine({
+                shape: {
+                    x1: pt1[0],
+                    y1: pt1[1],
+                    x2: pt2[0],
+                    y2: pt2[1]
+                },
+                style: {
+                    lineWidth: lineStyle.lineWidth
+                }
+            }).shape,
             style: lineStyle,
             strokeContainThreshold: opt.strokeContainThreshold || 5,
             silent: true,
             z2: 1
         });
-        graphic.setSubPixelOptimizeLine(line);
         line.anid = 'line';
```

Comment:
I think we can use `graphic.subPixelOptimizeLine(line)` directly here.

Reply:
This has been removed. You seemed to comment on an old commit.